graphene_sqlalchemy/tests/test_batching.py

```diff
@@ -91,8 +81,28 @@ def is_sqlalchemy_version_less_than(version_string):
 
 def test_many_to_one(session_factory):
     session = session_factory()
-    make_fixture(session)
-    schema = get_schema(session)
+
+    reporter_1 = Reporter(
```

Comment:
Why did you "broke" the factorization of this setup code ?

Reply:
Because different tests needed different fixtures. It's easier this way to keep track of the state of a given test or to add a new test.